README.md

```diff
@@ -1,2 +1,14 @@
-# SocialMediaDB
-A social media database built for our 3rd semester course Data and Applications.
+# Social Media DB
+
+A simple database application designed with miniworld as a social media, with CLI interface, which allows users to perform all the tasks in an actual social media such as sharing posts, reacting and commenting to posts, update profile details, etc.
+
+## Steps to Run
+
+
+```shell 
+ mysql -h <local host> -u <username> --port=<port> -p > SocialMediaDB.sql
+ python3 app.py
```

Comment:
Shouldn't it be `-p < SocialMediaDB.sql` or will this also work?

Reply:
@pavani-17

Reply:
Oops... No it wont work... It sud be <